src/js/csv.js

```diff
@@ -1,19 +1,48 @@
-/* Copyright(c) 2017 Philip Mulcahy. */
+/* Copyright(c) 2017-2020 Philip Mulcahy. */
 /* jshint strict: true, esversion: 6 */
+/* jslint node:true */
 
 "use strict";
 
 import save_file from './save_file';
 
-function download(table) {
+function download(table, type) {
     const tableToArrayOfArrays = function(table) {
         const rows = table.rows;
         const result = [];
-        for (let i=0; i<rows.length; ++i) {
-            const cells = rows[i].cells;
-            const cell_array = [];
-            for (let j=0; j<cells.length; ++j) {
-                cell_array.push(cells[j].textContent);
+        for(let i=0; i<rows.length + ( type  ?  -1  :  0 ); ++i) {
+            let cells = rows[i].cells;
+            let cell_array = [];
+            for(let j=0; j<cells.length; ++j) {
+                let x = cells[j];
+                if (x.getAttribute("class").search("azad_numeric_no") == -1) {
+                    x = x.textContent.replace(/^([£$]|CAD|EUR|GBP) */, '');
```

Comment:
The currency pattern appears in order.js, table.js. We should replace it with a function in util.js
Just saying - no need to do it in this pr.

Reply:
Yes, had similar thought.